src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGalleryListView.tsx

```diff
@@ -0,0 +1,128 @@
+        selectableRow={{ onChange: handleInputChange }}
+      >
+        <DataListItem aria-labelledby={id} id={id}>
+          <DataListItemRow>
+            <Flex justifyContent={{ default: 'justifyContentFlexStart' }}>
+              <FlexItem>
+                {labels.includes('source') ? (
+                  <PlusCircleIcon size="md" color="blue" />
+                ) : (
+                  <PlusCircleIcon size="md" color="green" />
+                )}
+              </FlexItem>
+              <FlexItem>
+                <DataListItemCells
+                  dataListCells={[
+                    <DataListCell key="primary content">
+                      <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                          <Flex>
+                            <FlexItem>
+                              <strong
+                                className="pf-u-font-weight-bold"
+                                id="selectable-action-item1"
+                              >
+                                {title}
+                              </strong>
+                            </FlexItem>
+                            <FlexItem>
+                              <OutlinedQuestionCircleIcon
+                                href="#filled"
+                                color="grey"
+                              />
+                            </FlexItem>
+                            <FlexItem>
+                              <span className="pf-u-font-size-sm pf-u-color-400">
+                                {verison}
+                              </span>
+                            </FlexItem>
+                          </Flex>
+                        </FlexItem>
+                        <FlexItem>
+                          <LabelGroup numLabels="10">
+                            {labels.includes('source') ? (
+                              <Label color="blue">Source</Label>
+                            ) : (
+                              <Label color="green">Sink</Label>
+                            )}
+                            {labels
+                              .filter(
+                                (label) =>
+                                  label !== 'source' && label !== 'sink'
+                              )
+                              .sort((a, b) =>
+                                a
+                                  .toLocaleLowerCase()
+                                  .localeCompare(b.toLocaleLowerCase())
+                              )
+                              .map((label) => (
+                                <Label key={label}>{t(label)}</Label>
+                              ))}
+                          </LabelGroup>
+                        </FlexItem>
+                      </Flex>
+                    </DataListCell>,
+                  ]}
+                />
+              </FlexItem>
+            </Flex>
+            <DataListAction
```

Comment:
This is one of the compile errors, I suggest though maybe just leave this commented out, as I wouldn't mind maybe adding a storybook where there's some buttons here to see how that looks.